player/src/main/java/com/focusit/jsflight/player/webdriver/SeleniumDriver.java

```diff
@@ -30,22 +31,33 @@
 
 /**
  * Selenium webdriver proxy: runs a browser, sends events, make screenshots
- * 
+ *
  * @author Denis V. Kirpichenkov
  *
  */
 public class SeleniumDriver
 {
 
+    /**
+     * Non operational element indicating step processing must be aborted
+     * returned from weblookup script
+     */
+    public static final RemoteWebElement NO_OP_ELEMENT = new RemoteWebElement();
     private static final Logger LOG = LoggerFactory.getLogger(SeleniumDriver.class);
-    private HashMap<String, WebDriver> drivers = new HashMap<>();
+    private static HashMap<String, WebDriver> drivers = new HashMap<>();
```

Comment:
why drivers should be a static field? Remember each time you create a static field one (or more) unit tests dies!